core/js/tests/specs/setupchecksSpec.js

```diff
@@ -1403,7 +1403,7 @@ describe('OC.SetupChecks tests', function() {
 
 			async.done(function( data, s, x ){
 				expect(data).toEqual([{
-					msg: 'It seems like you are running a 32bit PHP version. Nextcloud needs 64bit to run well. Please upgrade your OS and PHP to 64bit!',
+					msg: 'It seems like you are running a 32bit PHP version. Nextcloud needs 64bit to run well. Please upgrade your OS and PHP to 64bit! For further details read <a target="_blank" rel="noreferrer noopener" class="external" href="https://docs.example.org/admin-system-requirements">the documentation page about this ↗</a>',
```

Comment:
Not sure why the test fails... this should in theory return exactly this result...

Reply:
@MichaIng do you have any idea why this test seems to fail?

Reply:
It somehow get this:
```suggestion
					msg: 'It seems like you are running a 32bit PHP version. Nextcloud needs 64bit to run well. Please upgrade your OS and PHP to 64bit! For further details read <a href="https://docs.example.org/admin-system-requirements" class="external" rel="noreferrer noopener">the documentation page about this ↗</a>.',
```
But strangely all other checks get the attributes the other way round (+`target="_blank"`) and do not fail 🤔.

Reply:
lets try this then 👍